### PR TITLE
docs: trpc docs v0

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/astro.config.mjs [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default defineConfig({
   site: `https://beta.create.t3.gg`,
   markdown: {
     shikiConfig: {
-      theme: "material-palenight",
+      theme: "nord",
```
Comment (juliusmarminge) [inline www/astro.config.mjs:13]: subject to change, the `material-palenight` didn't look very good once some actual code was used 🥲 
image

File: www/src/pages/en/usage/trpc.md [hunk 2]
```diff
@@ -1,7 +1,114 @@
...
+    where: {
+      id,
+    },
+  });
+
+  res.status(200).json(examples);
+};
+
+export default userByIdHandler;
+```
```
Comment (juliusmarminge) [inline www/src/pages/en/usage/trpc.md:114]: probably won't stick around

**Comment by nexxeln** [comment]
Sorry that was vimium being weird.

**Comment by juliusmarminge** [comment]
First iteration, will continue later but can merge this for now I think.

**Review by nexxeln — COMMENTED** [review]
I think its worth adding a paragraph below explain why trpc is superior to nextjs api routes, like you don't have to declare types anymore etc

File: www/src/pages/en/usage/trpc.md [hunk 3]
```diff
@@ -1,7 +1,121 @@
...
+
+  return (
+    <div>
+      <h1>{user.data?.name}</h1>
+    </div>
+  );
+};
+```
+
+You'll immediately notice how good the autocompletion and typesafety is. Immediately when you have written `trpc.` your routers will show up, and eventually so will your procedures. You'll also get some red squiggly lines if your input doesn't match your validator that you defined on the backend.
```
Comment (nexxeln) [inline www/src/pages/en/usage/trpc.md:82]: Instead of red squiggly lines we should say typescript errors as red squiggly lines aren't in all editors

**Review by c-ehrlich — COMMENTED** [review]
Here are my suggestions :)

https://docs.google.com/document/d/1Mjd_2cULiZToRtQap6hVO2kz8xvhhRYoS865LoNSLpA/edit?usp=sharing

File: www/src/pages/en/usage/trpc.md [hunk 4]
```diff
@@ -1,7 +1,121 @@
 ---
-title: TypeScript
-description: Usage of TypeScript
+title: TRPC
```
Comment (nexxeln) [inline www/src/pages/en/usage/trpc.md:2]: Header should be tRPC not TRPC right?

**Review by nexxeln — COMMENTED** [review]
Also Typescript should be TypeScript.

**Comment by juliusmarminge** [comment]
Approve and ship

**Review by c-ehrlich — APPROVED** [review]

**Pull request merged**